interfaces/builtin/u2f_devices.go

```diff
@@ -102,6 +102,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "20a0",
 		ProductIDPattern: "4287",
 	},
+	{
+		Name:             "Nitrokey FIDO2",
+		VendorIDPattern:  "20a0",
+		ProductIDPattern: "42b1",
```

Comment:
lgtm, this matches what the nitrokey firmware says upstream: https://github.com/Nitrokey/nitrokey-fido2-firmware/commit/d62d9e6bcb9aa278e9d7f23762bb071299e68b16